src/main/java/harper/tasks/Deadline.java

```diff
@@ -0,0 +1,28 @@
+package harper.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline task.
+ */
+public class Deadline extends Task {
+    private LocalDateTime by;
+
+    /**
+     * Creates a task.Deadline task with description and deadline.
+     *
+     * @param description The description of the task.Deadline task.
+     * @param by The deadline of the task.
```

Comment:
Should this javadoc  be cleaned up? Seems like a relic from intellij auto refactor.

I see this issue in other places